src/components/dialog-full-screen/dialog-full-screen.stories.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { State, Store } from '@sambego/storybook-state';
+import notes from './notes.md';
+import DialogFullScreen from './dialog-full-screen';
+import Button from '../button';
+
+const store = new Store({
+  open: false
+});
+
+const handleCancel = () => {
+  store.set({ open: false });
+  action('cancel')();
+};
+const handleOpen = () => {
+  store.set({ open: true });
+  action('open')();
+};
+
+storiesOf('Dialog Full Screen', module)
+  .addParameters({
+    info: {
+      propTablesExclude: [Button, State]
+    }
+  })
+  .add('default', () => {
+    const title = text('title', 'Example Dialog');
+    const subtitle = text('subtitle', 'Example Subtitle');
+    const children = text('children', 'Text Content');
+    const enableBackgroundUI = boolean('enableBackgroundUI', false);
+    const disableEscKey = boolean('disableEscKey', false);
+    const ariaRole = text('ariaRole', 'dialog');
+
+    return (
+      <div>
+        <Button onClick={ handleOpen }>Open Preview</Button>
+        <State store={ store }>
+          <DialogFullScreen
+            open={ store.get('open') }
```

Comment:
I did have that originally, but you ended up either opening a blank page (Dialog not open) or with the Dialog open and the content behind it completely covered. 

In the end, I decided it made more sense to have a button for opening the Dialog.

Happy to poll opinions on this though.

Reply:
Ok so what do you think about using knob button instead of html button 
![zrzut ekranu 2019-03-6 o 11 06 15](https://user-images.githubusercontent.com/12142840/53873159-059f1480-4000-11e9-95ce-956ae5f03788.png)

Reply:
I think it's the same sort of issue to be honest; I don't want to hide an important bit of UI away in the Storybook tabs.

Reply:
Let's stick with state for now.